### PR TITLE
es & cqrs

### LINKED ISSUE #24: es & cqrs

### PR DESCRIPTION
**Pull request by eyakauleva**
close #24

### CODE REVIEW

File: src/main/java/com/solvd/micro9/users/web/dto/EsDto.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+@Data
+@NoArgsConstructor
+public class EsDto {
+
+    private Long id;
+
+    private String type;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm")
```
Comment (h1alexbel) [inline src/main/java/com/solvd/micro9/users/web/dto/EsDto.java:17]: line is too long

File: src/main/java/com/solvd/micro9/users/persistence/eventstore/listener/EsUserListener.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+@RequiredArgsConstructor
+public class EsUserListener extends AbstractMongoEventListener<EsUser> {
+
+    private final SequenceGeneratorService sequenceGenerator;
+
+    @Override
+    public void onBeforeConvert(BeforeConvertEvent<EsUser> event) {
+        try {
+            if (Objects.isNull(event.getSource().getId())) {
+                event.getSource().setId(sequenceGenerator.generateSequence(EsUser.SEQUENCE_NAME));
```
Comment (h1alexbel) [inline src/main/java/com/solvd/micro9/users/persistence/eventstore/listener/EsUserListener.java:24]: line is too long

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/main/java/com/solvd/micro9/users/web/dto/EsDto.java (hunk 1 above)
Comment (eyakauleva) [reply src/main/java/com/solvd/micro9/users/web/dto/EsDto.java]: fixed 6dd736b

File: src/main/java/com/solvd/micro9/users/persistence/eventstore/listener/EsUserListener.java (hunk 2 above)
Comment (eyakauleva) [reply src/main/java/com/solvd/micro9/users/persistence/eventstore/listener/EsUserListener.java]: fixed 6dd736b

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**